java/ql/src/semmle/code/java/frameworks/javase/Http.qll

```diff
@@ -0,0 +1,20 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+
+/* A class representing Http Request Builder. */
```

Comment:
Use backticks and the proper Java class name in doc comments (i.e., "representing `HttpRequest.Builder`")

Reply:
Also regular comments -> doc comments throughout